app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,9 @@
   </head>
 
   <body>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
+
```

Comment:
checked while re-watching Sat's lecture. Looks good!

Reply:
Cool, think that's been added now, I'll commit again